projects/aca-content/package.json

```
@@ -5,10 +5,10 @@
   "peerDependencies": {
     "@angular/common": ">=15.2",
     "@angular/core": ">=15.2",
-    "@alfresco/adf-core": ">=7.0.0-alpha.6",
-    "@alfresco/adf-content-services": ">=7.0.0-alpha.6",
-    "@alfresco/adf-extensions": ">=7.0.0-alpha.6",
-    "@alfresco/js-api": ">=8.0.0-alpha.6",
+    "@alfresco/adf-core": ">=7.0.0-alpha.6-0",
```

```suggestion
    "@alfresco/adf-core": ">=7.0.0-alpha.7-0",
```
This will be next alphas for ADF, some for others

---

I should also update the libs in the main package.json?

---

No, upstream will handle this, the crucial part was updating peerDeps to correct ADF and JS-API version